packages/aws-cdk-lib/custom-resources/lib/provider-framework/provider.ts

```diff
@@ -8,6 +8,7 @@ import * as ec2 from '../../../aws-ec2';
 import * as iam from '../../../aws-iam';
 import * as kms from '../../../aws-kms';
 import * as lambda from '../../../aws-lambda';
+import { determineLatestNodeRuntime } from '../../../aws-lambda/lib/runtime-determiner-lambda.generated';
```

Comment:
Explain to me why this code needed to be generated and couldn't just be a single handwritten function?

As far as I can tell, there are no parameters that go into the generation?

Reply:
Going to hand write the two functions `determineLatestNodeRuntime` and `determineLatestNodeRuntimeName` as discussed offline.